app/src/main/java/com/lumisdinos/gameoffifteen/ui/dialog/SimpleAlertDialog.kt

```diff
@@ -0,0 +1,46 @@
+package com.lumisdinos.gameoffifteen.ui.dialog
+
+import android.app.AlertDialog
+import android.content.Context
+import com.lumisdinos.gameoffifteen.R
+import com.lumisdinos.gameoffifteen.domain.repos.PuzzleLogicRepository.Companion.ACTION_CONGRATULATIONS
+
+fun getAlertDialog(
```

Comment:
Why not using normal alert in Fragment? It looks much cleaner
context?.let {
            MaterialAlertDialogBuilder(it)
                .setTitle("Title")
                .setMessage("Message")
                .setPositiveButton(getString(R.string.delete)) { _, _ ->
                  positiveClick()
                }
                .setNegativeButton(getString(R.string.cancel), null)
                .show()
        }

Reply:
Or move it to base fragment 
![base fragment](https://user-images.githubusercontent.com/13872254/99136279-c7465f00-2602-11eb-95cc-7a70bfede3c0.png)

Reply:
it's just suggestions, your solution is not bad

Reply:
I did it on your MaterialAlertDialogBuilder suggestion as in this particular the dialog is simple one